packages/hoppscotch-common/src/modules/dioc.ts

```diff
@@ -34,5 +35,8 @@ export default <HoppModule>{
     app.use(diocPlugin, {
       container: serviceContainer,
     })
+    for (const service of platform.addedServices ?? []) {
```

Comment:
Could you elaborate on the purpose of this new field?

Reply:
There can be additional services in specific platforms that are not present in common. Hence such additional services can be bound to the service container.

Eg. Licensing service is unique to selfhost-web and is not present in common. That can be bound using this.